src/main/frontend/src/components/WordShow.js

```diff
@@ -0,0 +1,31 @@
+import React, {useState, useEffect } from 'react'
+import ReviewForm from './ReviewForm'
+
+const WordShow = (props) => {
+  let wordId = props.match.params.id
+  const [pet, setPet] = useState([])
+  const [newReview, setNewReview] = useState([])
+
+  useEffect(() => {
+   fetch('api/v1/${props.location.pathname}')
```

Comment:
I'm thinking this is going to be the URL. @MtnGreen confirm?
```suggestion
   fetch('api/v1/words/${props.location.pathname}')
```

Reply:
Nevermind, my bad!